src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -1,17 +1,72 @@
 from __future__ import absolute_import
 
+import math
+from datetime import datetime
 from copy import deepcopy
 
 from rest_framework import status
 from rest_framework.response import Response
 
+from django.utils import timezone
+
 from sentry import features
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.paginator import OffsetPaginator
-from sentry.api.serializers import serialize
+from sentry.api.serializers import serialize, AlertRuleSerializer, CombinedRuleSerializer
 from sentry.incidents.models import AlertRule
-from sentry.incidents.endpoints.serializers import AlertRuleSerializer
+from sentry.models import Rule, RuleStatus
+from sentry.utils.cursors import build_cursor, Cursor
+
+
+class ProjectCombinedRuleIndexEndpoint(ProjectEndpoint):
```

Comment:
We need a test for this endpoint, which should also cover pagination

Reply:
On it.